action.yml

```
@@ -17,8 +17,8 @@ inputs:
     description: 'Also look for tags to delete'
     required: false
     default: false
-  include_prefixes:
-    description: 'List of branch prefixes to be deleted'
+  exclude_prefixes:
+    description: 'List of branch prefixes to exclude from deletion'
```

@hwoarang I was thinking if we could improve the description message a bit. For example:
```suggestion
    description: 'A comma-separated list of branch prefixes to exclude from deletion'
```

---

done